x-pack/test/idp-fixture/Vagrantfile

```diff
@@ -10,15 +10,20 @@ Vagrant.configure("2") do |config|
     config.cache.scope = :box
   end
 
-  config.vm.network "forwarded_port", guest: 389, host: 60389, protocol: "tcp"
-  config.vm.network "forwarded_port", guest: 636, host: 60636, protocol: "tcp"
```

Comment:
This one is used in x-pack/qa/openldap-tests/src/test/java/org/elasticsearch/test/OpenLdapTests.java